### PR TITLE
Update docs aws terraform ami example to 4.3.5 from 4.2.3

### PR DESCRIPTION
**Pull request by stevenGravy**
updating ami version as done in the github README for the AWS terraform example.

### CODE REVIEW

File: docs/4.3/aws-terraform-guide.md [hunk 1]
```diff
@@ -154,11 +154,11 @@ cluster from scratch, so choose carefully.
 
 ### ami_name
 
-Setting `export TF_VAR_ami_name="gravitational-teleport-ami-ent-4.2.3"`
+Setting `export TF_VAR_ami_name="gravitational-teleport-ami-ent-4.3.5"`
```
Comment (webvictim) [inline docs/4.3/aws-terraform-guide.md:157]:
```suggestion
Setting `export TF_VAR_ami_name="gravitational-teleport-ami-ent-{{ teleport.version }}"`
```

File: docs/4.3/aws-terraform-guide.md [hunk 2]
```diff
@@ -154,11 +154,11 @@ cluster from scratch, so choose carefully.
...
 
 Gravitational automatically builds and publishes OSS, Enterprise and Enterprise FIPS 140-2 AMIs when we
 release a new version of Teleport. The AMI names follow the format: `gravitational-teleport-ami-<type>-<version>`
-where `<type>` is either `oss` or `ent` (Enterprise) and `version` is the version of Teleport e.g. `4.2.3`.
+where `<type>` is either `oss` or `ent` (Enterprise) and `version` is the version of Teleport e.g. `4.3.5`.
```
Comment (webvictim) [inline docs/4.3/aws-terraform-guide.md:161]:
```suggestion
where `<type>` is either `oss` or `ent` (Enterprise) and `version` is the version of Teleport e.g. `{{ teleport.version }}`.
```

**Review by webvictim — COMMENTED** [review]
The latest Teleport version is actually stored as a variable in the mkdocs config already: https://github.com/gravitational/teleport/blob/master/docs/4.3.yaml#L36

We should be able to make use of this here so that it doesn't need updating on every release.

@benarent I think this is how it works?

**Review by webvictim — CHANGES_REQUESTED** [review]
Requesting changes to remove from review queue

File: docs/4.3/aws-terraform-guide.md (hunk 1 above)
Comment (benarent) [reply docs/4.3/aws-terraform-guide.md:157]: Yeah, that'll work!

**Review by benarent — APPROVED** [review]

**Comment by webvictim** [comment]
retest this please

**Review by webvictim — APPROVED** [review]

**Pull request merged**